sormas-ui/src/main/java/de/symeda/sormas/ui/samples/sampleLink/SampleListComponentLayout.java

```diff
@@ -11,14 +11,13 @@
 
 public class SampleListComponentLayout extends SideComponentLayout {
 
-	public SampleListComponentLayout(SampleListComponent sampleListComponent, String infoText) {
+	public SampleListComponentLayout(boolean hideInfoText, SampleListComponent sampleListComponent, String infoText) {
```

Comment:
please make infoText nullable instead of adding a flag to hide it

Reply:
removed the flag